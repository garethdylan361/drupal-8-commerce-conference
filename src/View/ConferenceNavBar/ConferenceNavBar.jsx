import React from 'react';
import Nav from 'react-bootstrap/lib/Nav';
import NavItem from 'react-bootstrap/lib/NavItem';

export default class ConferenceNavBar extends React.Component {

  constructor(props){
    super(props);
  }

  handleSelect(eventKey) {
    event.preventDefault();
    alert(`selected ${eventKey}`);
  }

  render(){

    return(
      <Nav bsStyle="tabs" activeKey="1" onSelect={this.handleSelect}>
        <NavItem eventKey="1" href="#">Analytics</NavItem>
        <NavItem eventKey="2" href="#">Manage</NavItem>
        <NavItem eventKey="3" href="#">Registration</NavItem>
        <NavItem eventKey="3" href="#">Name Tags</NavItem>
      </Nav>
    );
  }

}

