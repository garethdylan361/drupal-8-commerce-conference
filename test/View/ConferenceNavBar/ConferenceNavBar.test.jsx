import React from 'react';
import renderer from 'react-test-renderer';
import ConferenceNavBar from '../../../src/View/ConferenceNavBar/ConferenceNavBar';

test('ConferenceNavBar should render', () => {
  const navBar = renderer.create(<ConferenceNavBar/>);
  expect(navBar).toMatchSnapshot();
});
