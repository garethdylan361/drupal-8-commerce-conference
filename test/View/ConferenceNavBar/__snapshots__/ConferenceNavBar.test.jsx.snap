// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConferenceNavBar should render 1`] = `
<ul
  className="nav nav-tabs"
  role={null}
>
  <li
    className="active"
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Analytics
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Manage
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Registration
    </a>
  </li>
  <li
    className=""
    role="presentation"
    style={undefined}
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      Name Tags
    </a>
  </li>
</ul>
`;
